app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -62,6 +64,20 @@
     private ViewGroup.LayoutParams cParams;
     private Integer radius;
 
+    private static ArrayList<String> NETWORK_UNAVAILABLE = new ArrayList<>(
+            Arrays.asList(
+                    "You know, life without Wi-Fi is hard.",
+                    "maybe you should move to Canada, we have really good internet here :o)",
+                    "Pay me $50 and I will get you Wi-Fi",
+                    "I agree that free Wi-Fi shouldn't be forbidden",
+                    "Keep Looking on the internet. Surely there must be a good forecast somewhere" +
+                            " out there.",
+                    "I won't work until you provide me with life-long free Wi-Fi"
+            )
+    );
+
+    private static final String couldntHearYou = "Sorry, what was that? I didn't hear you.";
```

Comment:
Where does the callback for not saying anything happen? How are we currently handling it?